### PR TITLE
Create accessibility-how-to.md

### PR DESCRIPTION
**Pull request by hhartingrex**
@mgifford Here is a starting place. Once we get this page working - first draft - I will adjust the left navigation bar. Thank you!

----
:books: Documentation preview :books:: https://civicactions-handbook--1086.org.readthedocs.build/en/1086/

### CODE REVIEW

**Comment by hhartingrex** [comment]
Looks like I need to get the menu change in for this PR to be accepted

**Review by dmundra — COMMENTED** [review]
@hhartingrex do you want to deploy a work in progress page or hold off till it has more content in it?

**Comment by hhartingrex** [comment]
Thank you @dmundra . 
@mgifford is our handoff. He is going to take this and spiff it up, then we will publish first draft. Thanks Mike!

File: practice-areas/accessibility-how-to.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Accessibility: How To
+
+## Resources for learning how to test the accessibility of our applications
```
Comment (mgifford) [inline practice-areas/accessibility-how-to.md]: @maggiewachs what do you think of this extension?

Comment (mgifford) [inline practice-areas/accessibility-how-to.md]: @maggiewachs what do you think of this extension?

Comment (mgifford) [inline practice-areas/accessibility-how-to.md]: @maggiewachs what do you think of this extension?

File: practice-areas/accessibility-how-to.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Most operating systems have a screen reader built in. One of the most popular for blind users is JAWS, but the expense and customizability presents limitations for users. Most casual accessibility testers will leverage the built-in VoiceOver screen reader on a Mac and the open source [NVDA on Windows](https://www.nvaccess.org/download/) for most testing. If using NVDA, consider contributing from your ProDev to support this great open source project. According to a 2021 by [WebAim survey of screen reader users](https://webaim.org/projects/screenreadersurvey9/), ChromeVox & Windows Narrator combined make up less than 1% of use. It is also worth noting that testing on mobile devices is important and that both iOS (VoiceOver) and Android (TalkBack) have built-in screen readers.
+
+It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn to empathize with blind users, as they are navigating through the page.
```
Comment (maggiewachs) [inline practice-areas/accessibility-how-to.md]: This is admittedly a nit-picky comment :) But I like to reframe empathy as needing to learn from screen reader users' expertise. I'd change this to: "It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA)."

File: practice-areas/accessibility-how-to.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+How to test: Apple has produced a great quantity of documentation on both the [MacOSX](https://support.apple.com/en-gb/guide/voiceover/welcome/mac) & [iOS](https://support.apple.com/en-gb/guide/iphone/iph3e2e415f/ios) versions of VoiceOver. WebAim also has useful resources for [MacOSX](https://webaim.org/articles/voiceover/) & [iOS](https://webaim.org/articles/voiceover/mobile) versions. Harvard University published this [guide on using NVDA](https://accessibility.huit.harvard.edu/nvda) and [WebAim also has a similar guide](https://webaim.org/articles/nvda/).
+
+Helpful tips: Make sure you know how to turn it off before you turn it on.
```
Comment (maggiewachs) [inline practice-areas/accessibility-how-to.md]: Excellent point!

**Review by maggiewachs — COMMENTED** [review]
This is great, ship it!

File: practice-areas/accessibility-how-to.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Accessibility: How To
+
+## Resources for learning how to test the accessibility of our applications
+
+It is worth looking over the [resources on testing on Drupal.org](https://www.drupal.org/docs/accessibility/how-to-do-an-accessibility-review). The following resources are a more consolidated list put in order of priority. They are structured from lowest to greatest effort, and addressing issues in steps 1 & 2 will make screen reader and voice control testing easier:
+
+1/ [Accessibility Insights](https://accessibilityinsights.io/) (FastPass)
+
+Microsoft's open source Accessibility Insights offers both the automated testing with axe and guided keyboard only testing within their FastPass. This is roughly equivalent to using WebAim's proprietary [Wave](https://wave.webaim.org/) toolbar with \[Keyboard-only testing\](https://webaim.org/techniques/keyboard/). The main advantage of Accessibility Insights is that it provides assets which make it easier to include in JIRA to replicate the results.
```
Comment (dmundra) [inline practice-areas/accessibility-how-to.md]: @mgifford looks like this link `\[Keyboard-only testing\](https://webaim.org/techniques/keyboard/)` is being escaped.

**Review by dmundra — CHANGES_REQUESTED** [review]
@mgifford looking good, one minor link fix and then this can be merged.

**Comment by ContentBrewmaster** [comment]
Before we merge this, let's consider a couple of things:
- we now have 2 pages for the Accessibility practice area, so they need their own folder
- the steps should be change to H3 so they show up in the "on this page" nav on the right

**Comment by hhartingrex** [comment]
@ContentBrewmaster Can you do the folder for us? I can change the page to have the H3s. Thank you!

File: practice-areas/accessibility-how-to.md (hunk 4 above)
Comment (mgifford) [reply practice-areas/accessibility-how-to.md]: I think I squashed that now.

**Comment by hhartingrex** [comment]
@dmundra Are there any more requested changes that we have not met? I think we are ready! THANK YOU for help (as usual!)

**Comment by dmundra** [comment]
> @dmundra Are there any more requested changes that we have not met? I think we are ready! THANK YOU for help (as usual!)

@hhartingrex can you take a look at #1092. It will have the folder changes requested for this branch.

**Review by ContentBrewmaster — COMMENTED** [review]
@dmundra @maggiewachs there seem to be unresolved comments in this content. Can you take a look?

**Review by dmundra — COMMENTED** [review]
@hhartingrex, @mgifford FYI I merged my folder changes for the accessibility documents into this PR.

**Review by dmundra — APPROVED** [review]
Changes look good to me! @mgifford can you review the comment from Maggie?

File: practice-areas/accessibility-how-to.md (hunk 2 above)
Comment (mgifford) [reply practice-areas/accessibility-how-to.md]: That makes sense to me. Like that you include a link to a Smashing video. So just replace that sentence with line 25 or just part of it?

Comment (ContentBrewmaster) [reply practice-areas/accessibility-how-to.md]: @maggiewachs can you follow up on Mike's suggestion so we can merge this PR?

**Comment by ContentBrewmaster** [comment]
@mgifford can you look at Maggie's comment so we can merge this PR?

File: practice-areas/accessibility-how-to.md (hunk 2 above)
Comment (mgifford) [reply practice-areas/accessibility-how-to.md]: Ok, I've modified it. Now reads as:  
  
`It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA).`

**Comment by dmundra** [comment]
@maggiewachs, @hhartingrex, @mgifford we merged these changes since we have a new feature coming down soon (#1123). Once that is merged, feel free to make adjustments to changes introduced here.

**Pull request merged**